vocabularies/Analytics.json

```diff
@@ -161,6 +164,102 @@
         "@Validation.ApplicableTerms@Core.Description": "Adding a list of other terms that can be annotated to it.",
         "@Validation.ApplicableTerms": ["Common.Label"]
       }
+    },
+    "AutoExpand": [
+      {
+        "$Kind": "Function",
+        "$EntitySetPath": "InputSet",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.Description": "`$apply` transformation that expands an unnamed leveled hierarchy with custom aggregation of certain properties",
+        "@Core.LongDescription": "Example transformation sequence:\n```\n$apply=filter(Industry eq 'IT')\n/groupby((Country,CountryName,Region,RegionName),\n         filter($these/aggregate(Amount) gt 0 and\n                $these/aggregate(Currency) ne null))\n/concat(\n  Analytics.AutoExpand(\n    Levels=[{\"P\":[\"Country\",\"CountryName\"]},\n            {\"P\":[\"Region\",\"RegionName\"]}],\n    LeafLevel=[\"Segment\",\"Industry\"],\n    Aggregation=[\"Amount\",\"Currency\"],\n    SiblingOrder=[{\"Property\":\"Amount\",\n                   \"Order\":\"desc\"}],\n    ExpandLevels=[{\"Entry\":[\"US\",\"USA\"],\"Levels\":0},\n                  {\"Entry\":[\"DE\",\"Germany\",\"BW\",\"Baden-Württemberg\"],\"Levels\":2}])\n    /concat(aggregate($count) as ResultEntriesCount,\n            skip(20)/top(10)),\n  aggregate(Amount,Currency))\n```\n",
```

Comment:
In UI5 we start with the leaves count that is missing in the example. It would be a term `groupby((Country,CountryName,Region,RegionName))/aggregate($count as LeavesCount)`  in the outer concat followed by the grand total term `aggregate(Amount,Currency)` that is currently last in the example. The third term in the outer concat would be `Analytics.AutoExpand(...)/concat(aggregate($count) as ResultEntriesCount,skip(20)/top(10))`.